.github/workflows/main.yml

```diff
@@ -20,10 +20,10 @@ jobs:
         run: |
           if [ -z $ACT ]
           then 
-            _ver="[3.8, 3.9]"
+            _ver="['3.9', '3.10']"
             _cache="1"
           else 
-            _ver="[3.8]"
+            _ver="['3.10']"
```

Comment:
leave to 3.8 - it's where the most errors will likely pop, stuff like type annotations. 3.10 is what probably people have already, so they can just run pytest and whatnot by hand.